### PR TITLE
split Merged tables into manifest CSVs by grant number

### PR DESCRIPTION
**Pull request by vpchung**

### CODE REVIEW

File: python/data_model.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    ],
+    'project': [
+        'projectId', 'themeId', 'consortiumId', 'grantId', 'grantName', 'grantType'
+    ]
+}
+
+# Missing columns in the table schema that exist in the data model.
+ADD = {
+    'publication': [
+        {'index': 15, 'colname': 'Publication Tool Name', 'value': ""},
```
Comment (brynnz22) [inline python/data_model.py]: I took out Publication Tool Name from the data model, so I think we can remove this? Unless it does not make a difference to be in here.

File: python/data_model.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        {'index': 0, 'colname': 'Component', 'value': "PublicationView"},
+    ],
+    'dataset': [
+        {'index': 12, 'colname': 'Dataset Tissue', 'value': ""},
+        {'index': 0, 'colname': 'Component', 'value': "DatasetView"},
+    ],
+    'tool': [
+        {'index': 2, 'colname': 'Tool Dataset Alias', 'value': ""},
```
Comment (brynnz22) [inline python/data_model.py]: I also removed Tool Dataset Alias from the data model

**Review by brynnz22 — CHANGES_REQUESTED** [review]
Hey! I just made a couple of suggestions to remove the ADD columns that I already removed from the data model. Otherwise, it looks good to me!

File: python/data_model.py (hunk 1 above)
Comment (vpchung) [reply python/data_model.py]: Ah right, good catch!

File: python/data_model.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        'consortium': 'Dataset Consortium Name',
+        'theme': 'Dataset Theme Name',
+        'datasetName': 'Dataset Name',
+        'datasetAlias': 'Dataset Alias',
+        'description': 'Dataset Description',
+        'overallDesign': 'Dataset Design',
+        'assay': 'Dataset Assay',
+        'species': 'Dataset Species',
+        'tumorType': 'Dataset Tumor Type',
+        'externalLink': 'Dataset External Link'
```
Comment (brynnz22) [inline python/data_model.py:41]: I think we can change `Dataset External Link` to `Dataset Url` and I will remember to change the link to the url. Although, I am not sure if this change is necessary. What do you think?

**Review by brynnz22 — CHANGES_REQUESTED** [review]

File: python/data_model.py (hunk 3 above)
Comment (vpchung) [reply python/data_model.py:41]: Yep, I've already changed this on my end :) Thanks for double-checking!

Comment (brynnz22) [reply python/data_model.py:41]: Great!

**Review by brynnz22 — APPROVED** [review]
Okay, I think that's it for data model adjustments.

**Pull request merged**